worker/src/ocall_bridge/bridge_api.rs

```diff
@@ -96,6 +121,7 @@ impl Into<sgx_status_t> for OCallBridgeError {
             OCallBridgeError::GetUpdateInfo(s) => s,
             OCallBridgeError::GetIasSocket(_) => sgx_status_t::SGX_ERROR_UNEXPECTED,
             OCallBridgeError::SendBlockAndConfirmation(_) => sgx_status_t::SGX_ERROR_UNEXPECTED,
+            OCallBridgeError::IpfsError(_) => sgx_status_t::SGX_ERROR_UNEXPECTED,
```

Comment:
I remember now, why `from_over_into` is a clippy lint. This has nothing to do with bidirectionality, but with the methods that are exposed.

If we implement `From<OcallBrigdeError>  for sgx_status_t` we get also `Into<sgx_status_t> for OcallBridgeError`

Reply:
I only understood this just recently. I didn't know you could `impl` something that is not in you own module (in this case we `impl` `From<>` for `sgx_status_t`. I will make that change, and then we can remove the clippy warning override.

Reply:
Usually, you can't do this, the rules regarding that are quite involved. In this case, however, it works because `From<>` is parametrized by a local type.